content/account/users.textile

```diff
@@ -0,0 +1,72 @@
+---
+title: User management
+meta_description: "Learn how to manage users, user roles, and the permissions associated with each role."
+meta_keywords: "users, user management, admin, owner, permission"
+---
+
+The user that creates an Ably account is assigned the account owner role. An account owner has permission to undertake any action within an account, such as inviting new users. There are two other account roles that inherit a subset of an account owner's permissions. An account may only have a single account owner.
+
+h2(#roles). User roles
+
+Users are assigned one of three roles:
+
+* Standard
+* Admin
+* Owner
+
+User roles have the following permissions:
+
+|_. Permission |_. Standard |_. Admin |_. Owner |
+| View all apps | ✓ | ✓ | ✓ |
+| View app configuration | ✓ | ✓ | ✓ |
+| View app settings | ✓ | ✓ | ✓ |
+| View "app statistics":/metadata-stats/stats#app | ✓ | ✓ | ✓ |
+| View "account statistics":/metadata-stats/stats#account | ✓ | ✓ | ✓ |
+| Configure own "2FA":/account/2fa | ✓ | ✓ | ✓ |
+| "Invite new users":#invite | - | ✓ | ✓ |
+| "Remove existing users":#remove | - | ✓ | ✓ |
+| Manage "API keys":/auth | - | ✓ | ✓ |
```

Comment:
Probably not needed yet, but once we add a Billing role it will be useful to show another line for `View API keys`, to show that Standard users can view them, and Billing users can not

Reply:
I think this would be a little confusing at the moment, but we can definitely add an additional line when another role is added.